omid/Dockerfile

```diff
@@ -0,0 +1,85 @@
+# syntax=docker/dockerfile:1
+FROM stackable/image/java-base AS builder
+
+ARG PRODUCT
+
+RUN microdnf update && \
+    microdnf install tar maven java-11-openjdk-devel && \
+    microdnf clean all
+
+USER stackable
+WORKDIR /stackable
+
+SHELL ["bash", "-euo", "pipefail", "-c"]
+
+RUN curl -L https://dist.apache.org/repos/dist/release/phoenix/phoenix-omid-${PRODUCT}/phoenix-omid-${PRODUCT}-src.tar.gz | tar -xzC .
```

Comment:
Please use `--fail` with curl (also below)

Reply:
`tar` should already fail then, but still worth catching.

Reply:
4bacb5adbbd030cea0e8361ef29567ca1304093f